src/pages/deploy/deployments.md

```diff
@@ -139,3 +143,7 @@ layout="responsive"
 width={947} height={156} quality={80} />
 
 Restarting a crashed Deployment restores the exact image containing the code & configuration of the original build. Once the Deployment is back online, its status will change back to `Success`.
+
+## How come my GitHub Repo wont deploy?
+
+Railway will not deploy any commit from a user who is not in your team or invited to your project without their associated GitHub account.
```

Comment:
This is only true for PR deploys. We will still deploy commits from non-Railway users to the main branch connected to a service.